src/constants.py

```diff
@@ -62,107 +72,101 @@
-        "energyVector",
-        "unit",
+    ENERGY_CONVERSION: [
+        AGE_INSTALLED,
+        CAPEX_FIX,
+        CAPEX_VAR,
+        EFFICIENCY,
+        INFLOW_DIRECTION,
+        INSTALLED_CAP,
+        LABEL,
+        LIFETIME,
+        OPEX_FIX,
+        OPEX_VAR,
+        OPTIMIZE_CAP,
+        OUTFLOW_DIRECTION,
+        OEMOF_ASSET_TYPE,
+        ENERGY_VECTOR,
+        UNIT,
     ],
-    "energyStorage": [
-        "inflow_direction",
-        "label",
-        "optimizeCap",
-        "outflow_direction",
-        "type_oemof",
-        "storage_filename",
-        "energyVector",
+    ENERGY_STORAGE: [
+        INFLOW_DIRECTION,
+        LABEL,
+        OPTIMIZE_CAP,
+        OUTFLOW_DIRECTION,
+        OEMOF_ASSET_TYPE,
+        STORAGE_FILENAME,
+        ENERGY_VECTOR,
     ],
-    "energyProduction": [
-        "age_installed",
-        "capex_fix",
-        "capex_var",
-        "file_name",
-        "installedCap",
-        "label",
-        "lifetime",
-        "opex_fix",
-        "opex_var",
-        "optimizeCap",
-        "outflow_direction",
-        "type_oemof",
-        "unit",
-        "energyVector",
+    ENERGY_PRODUCTION: [
+        AGE_INSTALLED,
+        CAPEX_FIX,
+        CAPEX_VAR,
+        FILENAME,
+        INSTALLED_CAP,
+        LABEL,
+        LIFETIME,
+        OPEX_FIX,
+        OPEX_VAR,
+        OPTIMIZE_CAP,
+        OUTFLOW_DIRECTION,
+        OEMOF_ASSET_TYPE,
+        UNIT,
+        ENERGY_VECTOR,
     ],
-    "energyProviders": [
+    ENERGY_PROVIDERS: [
         "energy_price",
         "feedin_tariff",
-        "inflow_direction",
-        "label",
-        "optimizeCap",
-        "outflow_direction",
-        "peak_demand_pricing",
-        "peak_demand_pricing_period",
-        "type_oemof",
-        "energyVector",
+        INFLOW_DIRECTION,
+        LABEL,
+        OPTIMIZE_CAP,
+        OUTFLOW_DIRECTION,
+        PEAK_DEMAND_PRICING,
+        PEAK_DEMAND_PRICING_PERIOD,
+        OEMOF_ASSET_TYPE,
+        ENERGY_VECTOR,
     ],
-    "fixcost": [
-        "age_installed",
-        "capex_fix",
-        "capex_var",
-        "label",
-        "lifetime",
-        "opex_fix",
-        "opex_var",
+    FIX_COST: [
+        AGE_INSTALLED,
+        CAPEX_FIX,
+        CAPEX_VAR,
+        LABEL,
+        LIFETIME,
+        OPEX_FIX,
+        OPEX_VAR,
     ],
-    "simulation_settings": [
-        "evaluated_period",
-        "label",
+    SIMULATION_SETTINGS: [
+        EVALUATED_PERIOD,
+        LABEL,
         "output_lp_file",
         "restore_from_oemof_file",
         "display_nx_graph",
         "store_nx_graph",
-        "start_date",
+        START_DATE,
         "store_oemof_results",
-        "timestep",
+        TIMESTEP,
     ],
-    "project_data": [
+    PROJECT_DATA: [
         "country",
-        "label",
+        LABEL,
         "latitude",
         "longitude",
         "project_id",
         "project_name",
         "scenario_id",
         "scenario_name",
```

Comment:
This shows which strings are not included in the constant variables yet

Reply:
Yes, I left them there as they are not very used yet and I had the idea this can be done next time someone use one of these variables :)

Reply:
42919f2dafeb859be2118108ade52a3d5840d5eb